src/definitions/api-class.ts

```diff
@@ -0,0 +1,61 @@
+import * as ts from "typescript";
+
+import { ApiItem, ApiItemOptions } from "../abstractions/api-item";
+import { Logger, LogLevel } from "../utils/logger";
+import { TSHelpers } from "../ts-helpers";
+import { ApiHelpers } from "../api-helpers";
+import { ApiClassDto } from "../contracts/definitions/api-class-dto";
+import { ApiItemReferenceDict } from "../contracts/api-item-reference-dict";
+import { ApiItemKinds } from "../contracts/api-item-kinds";
+import { TypeDto } from "../contracts/type-dto";
+
+export class ApiClass extends ApiItem<ts.ClassDeclaration, ApiClassDto> {
+    constructor(declaration: ts.ClassDeclaration, symbol: ts.Symbol, options: ApiItemOptions) {
+        super(declaration, symbol, options);
+
+        // Members
+        this.members = ApiHelpers.GetItemsFromDeclarationsIds(declaration.members, this.Options);
+
+        // Extends
+        if (declaration.heritageClauses != null) {
+            const extendingList = ApiHelpers.GetHeritageList(declaration.heritageClauses, ts.SyntaxKind.ExtendsKeyword, this.Options);
+
+            if (extendingList.length > 1) {
+                Logger.Log(LogLevel.Warning, "Extending class more than 1 is not supported!");

```

Comment:
Is it possible?

Reply:
In current TypeScript is not possible. Well on second though, when TypeScript will introduce this change we will update then too.